packages/app-graphql-playground/index.js

```diff
@@ -5,9 +5,14 @@ const falsey = require('falsey');
 const { addDevQueryMiddlewares } = require('./lib/devQuery');
 
 class GraphQLPlaygroundApp {
-  constructor({ apiPath = '/admin/api', graphiqlPath = '/admin/graphiql' } = {}) {
+  constructor({
+    apiPath = '/admin/api',
+    graphiqlPath = '/admin/graphiql',
+    disableLogging = falsey(process.env.DISABLE_LOGGING),
```

Comment:
`disableLogging` should be `enableLogging`. Similarly in `packages/keystone/lib/Keystone/index.js`

Reply:
This would be a change. Currently the default for logging is `true` do you want to make the default `false`?